visualization/main.py

```diff
@@ -39,6 +39,8 @@ def read_data(tool: str):
         )
 
         for metric, value in df.itertuples(index=False, name=None):
+            if tag_number == "flask-2.0.0rc1" or tag_number == "flask-2.0.0rc2":
+                continue
```

Comment:
I am assuming that this is a flask fix?

Reply:
Yes this doesn't need to be added to main